content/200-concepts/100-components/03-prisma-migrate/050-prisma-migrate-flows.mdx

```diff
@@ -1,244 +0,0 @@
----
```

Comment:
We are pointing to this removed page here 
https://github.com/prisma/pris.ly/blob/master/_redirects#L45

Could you update it after the merge / prepare a PR, I could also do it, I would need the new link.

Reply:
I'm adding redirects, fear not! 👍